### PR TITLE
Add * to all mandatory fields using base_helper

### PR DESCRIPTION
**Pull request by aishmita-aggarwal**

### CODE REVIEW

File: backend/app/helpers/spree/admin/base_helper.rb [hunk 1]
```diff
@@ -140,10 +140,18 @@ def plural_resource_name(resource_class)
         resource_class.model_name.human(count: I18N_PLURAL_MANY_COUNT)
       end
 
+      def required_title_for(title)
```
Comment (jitendra) [inline backend/app/helpers/spree/admin/base_helper.rb]: I think required_label_markup will be better name

File: backend/app/helpers/spree/admin/base_helper.rb [hunk 2]
```diff
@@ -140,10 +140,18 @@ def plural_resource_name(resource_class)
...
+        raw(title + required_tag)
+      end
+
       private
         def attribute_name_for(field_name)
           field_name.gsub(' ', '_').downcase
         end
+
+        def required_tag
```
Comment (jitendra) [inline backend/app/helpers/spree/admin/base_helper.rb]: required_markup

**Comment by jitendra** [comment]
Please change the names of the helper methods

**Comment by aishmita-aggarwal** [comment]
@jitendra Done with the naming fixes. Please check.

**Comment by jitendra** [comment]
:+1: Good work. Please create a direct PR

**Pull request closed without merging**